.github/workflows/release.yml

```diff
@@ -0,0 +1,104 @@
+on:
+  push:
+    tags:
+      - "v*" # push events to matching v*, i.e. v1.0, v20.15.10
+
+jobs:
+  completions-upload:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target: ["bash", "fish", "zsh"]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Copy completions
+        run: |
+          cp ${{ matrix.target }}_tealdeer completions_${{ matrix.target }}
+      - uses: softprops/action-gh-release@v1 # actions/create-release, actions/upload-release-asset is deprecated
```

Comment:
Huh, it's a bit disappointing that there's no official action for this type of thing 😕

Reply:
I was also embarrassed because it deprecated suddenly.
It's still an incomprehensible decision.